os/board/rtl8730e/src/component/bluetooth/example/ble_scatternet/scatternet.c

```diff
@@ -267,15 +267,28 @@ static rtk_bt_evt_cb_ret_t ble_tizenrt_scatternet_gap_app_callback(uint8_t evt_c
     }
 
 #if RTK_BLE_5_0_AE_SCAN_SUPPORT
-		case RTK_BT_LE_GAP_EVT_EXT_SCAN_RES_IND: {
-			rtk_bt_le_ext_scan_res_ind_t *scan_res_ind = (rtk_bt_le_ext_scan_res_ind_t *)param;
-			rtk_bt_le_addr_to_str(&(scan_res_ind->addr), le_addr, sizeof(le_addr));
-			printf("[APP] Ext Scan info, [Device]: %s, AD evt type: 0x%x, RSSI: %i, PHY: 0x%x, TxPower: %d, Len: %d\r\n", 
-					le_addr, scan_res_ind->evt_type, scan_res_ind->rssi,
-					(scan_res_ind->primary_phy << 4) | scan_res_ind->secondary_phy,
-					scan_res_ind->tx_power, scan_res_ind->len);
-			break;
-		}
+    case RTK_BT_LE_GAP_EVT_EXT_SCAN_RES_IND: {
+        rtk_bt_le_ext_scan_res_ind_t *scan_res_ind = (rtk_bt_le_ext_scan_res_ind_t *)param;
+        rtk_bt_le_addr_to_str(&(scan_res_ind->addr), le_addr, sizeof(le_addr));
+        // printf("[APP] Ext Scan info, [Device]: %s, AD evt type: 0x%x, RSSI: %i, PHY: 0x%x, TxPower: %d, Len: %d\r\n", 
+        // 		le_addr, scan_res_ind->evt_type, scan_res_ind->rssi,
+        // 		(scan_res_ind->primary_phy << 4) | scan_res_ind->secondary_phy,
+        // 		scan_res_ind->tx_power, scan_res_ind->len);
+        trble_scanned_device scanned_device;
+        scanned_device.adv_type = scan_res_ind->evt_type;
```

Comment:
typedef enum {
	TRBLE_ADV_TYPE_IND,
	TRBLE_ADV_TYPE_DIRECT,
	TRBLE_ADV_TYPE_SCAN_IND,
	TRBLE_ADV_TYPE_NONCONN_IND,
	TRBLE_ADV_TYPE_SCAN_RSP,
	TRBLE_ADV_TYPE_UNKNOWN,
} trble_adv_type_e;

the maximum value defined in `scanned_device.adv_type` seems to be 5. 
However, when comparing it in a conditional statement, 0x1b and 0x13 are larger values than 5.
Can you map the values properly?